pkg/apis/mesh/types.go

```diff
@@ -51,11 +51,11 @@ const (
 
 // NATS channel names.
 const (
-	// NatsAgentConfigChannel sends the mesh config from mesh-api to agent.
+	// NatsAgentConfigChannel sends the mesh config from mesh-controller to agent.
 	NatsAgentConfigChannel = "nginx.nsm.agent.config"
-	// NatsAgentSubChannel sends a subscription and version notice from agent to mesh-api.
+	// NatsAgentSubChannel sends a subscription and version notice from agent to mesh-controller.
 	NatsAgentSubChannel = "nginx.nsm.agent.subscription"
-	// NatsAPIPingChannel sends a ping from mesh-api to agent on restart.
+	// NatsAPIPingChannel sends a ping from mesh-controller to agent on restart.
 	NatsAPIPingChannel = "nginx.nsm.api.ping"
```

Comment:
One other option is to just support both channels, and then after 2.0 we delete the old channel.

Reply:
I'm okay doing that as well, although I guess we don't have to since we've marked this a breaking change. Anyone have a strong preference for one vs the other?

Reply:
Despite being a breaking release, our upgrade process still works, so I'd like to see this still work if possible.